enterprise/server/backends/prom/prom.go

```diff
@@ -104,11 +114,62 @@ func (c *bbMetricsCollector) Collect(out chan<- prometheus.Metric) {
 }
 
 func (q *promQuerier) FetchMetrics(ctx context.Context, groupID string) (model.Vector, error) {
+	resultVector, err := q.getCachedMetrics(ctx, groupID)
+	if err != nil {
+		log.Errorf("failed to get cached metrics: %s", err)
```

Comment:
I would use Warningf here and below. I generally reserve Error logs for more critical failures.